docs/rootfs-and-kernel-setup.md

```diff
@@ -119,9 +119,16 @@ Alpine Linux:
 
    # Then, copy the newly configured system to the rootfs image:
    for d in bin etc lib root sbin usr; do tar c "/$d" | tar x -C /my-rootfs; done
+
+   # This command may trigger the following message:
+   tar: Removing leading "/" from member names
```

Comment:
Let's also add this as a comment, since people might copy-paste it and it would be treated as a bash command

Reply:
Thank you for your suggestion but we have already said that  "the following message" is a warning message. and when they use the 'for d in bin etc lib root sbin usr; do tar c "/$d" | tar x -C /my-rootfs; done' command , they will be see ' tar: Removing leading "/" from member names' which is warning message at the terminal. I just want to add this idea :) "Hey! this message is a warning not a error and skip it and continue"

Reply:
I meant to simply add a `#` character to the line. Thanks Diana for helping here.